data/json/items/armor/bespoke_armor/custom_underwear.json

```diff
@@ -13,10 +13,10 @@
     "looks_like": "union_suit",
     "color": "white",
     "armor": [
-      { "covers": [ "torso" ], "coverage": 98, "cover_melee": 90, "encumbrance": 1 },
-      { "covers": [ "arm_l", "arm_r" ], "coverage": 98, "cover_melee": 90, "encumbrance": 1 },
-      { "covers": [ "leg_l", "leg_r" ], "coverage": 100, "cover_melee": 90, "encumbrance": 1 },
-      { "covers": [ "foot_l", "foot_r" ], "coverage": 100, "cover_melee": 90, "encumbrance": 1 }
+      { "covers": [ "torso" ], "coverage": 98, "cover_melee": 90, "encumbrance": 1, "breathability": "GOOD" },
```

Comment:
need to lint these, I bet with the additional entry they need to be split on multiple lines. Don't do it by hand copy the whole file into https://dev.narc.ro/cataclysm/format.html

Reply:
Thank you, I was suspecting that another tool was needed since I lost that "json-styled" tag